src/include/OpenImageIO/imagecache.h

```diff
@@ -724,42 +724,51 @@ class OIIO_API ImageCache {
                         int miplevel, ustring dataname, TypeDesc datatype,
                         void* data);
 
-    /// Copy the ImageSpec associated with the named image (the first
-    /// subimage & miplevel by default, or as set by `subimage` and
-    /// `miplevel`).
-    ///
+    /// Copy the ImageSpec that describes the named image file.
+    ///
+    /// Note that the spec returned describes the file as it exists in the
+    /// file, at the base (highest-resolution) MIP level of that subimage.
+    /// Certain aspects of the in-cache representation may differ from the
+    /// file (due to ImageCache implementation strategy or options like
+    /// `"forcefloat"` or `"autotile"`). If you really need to know the
+    /// in-cache data type, tile size, or how the resolution or tiling changes
+    /// on a particular MIP level, you should use `get_cache_dimensions()`.
```

Comment:
I'm not sure, but I think maybe this needs a blank line or the `@param` will get folded into that paragraph?

```suggestion
    /// on a particular MIP level, you should use `get_cache_dimensions()`.
    ///
```